Create _github-pages-challenge-wen0x0.wen.json

wen0x0 opened the pull request:
<!--
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, NONE OF IT IS OPTIONAL UNLESS SPECIFIED.
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->

<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [x] I have **read** and **agree** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied/mostly blank templated websites. -->
- [x] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [x] I have provided a preview of my website below. <!-- This step is required for your domain to be approved. -->

# Website Preview
<!-- Provide a link or screenshot of your website below. -->
https://wen0x0.github.io/

wen0x0 committed:
Create _github-pages-challenge-wen0x0.wen.json

STICKnoLOGIC commented:
@wen0x0 
you need to get first the subdomain ```"wen.json"``` before creating a nested subdomain.

you can refer and read here: https://docs.is-a.dev/guides/github-pages/

you don't need to close this PR, just create the wen.json and it will automatically sync in this PR.

Best Regards,
STICKnoLOGIC

wen0x0 committed:
Rename _github-pages-challenge-wen0x0.wen.json to wen.json

wen0x0 reviewed (commented):
File name updated. Thanks for your support!

STICKnoLOGIC commented:
> File name updated. Thanks for your support!
@wen0x0 
please read this documentation: https://docs.is-a.dev/guides/github-pages/
it will help you to setup your domain in github page since there are (a lot of) missing parts and json file/s.

Best Regards,
STICKnoLOGIC

wen0x0 committed:
Update wen.json

Hi, I have updated the records to use CNAME for GitHub Pages as instructed.
Please let me know if any further changes are needed.

wen0x0 committed:
Merge branch 'main' into main

dragsbruh commented on domains/wen.json:
```diff
@@ -0,0 +1,9 @@
+        "email": "quocanh010104@gmail.com"
+    },
+    "records": {
+        "cname": "wen0x0.github.io"
```
use uppercase `CNAME`

i highly recommend to read the docs: https://docs.is-a.dev/domain-structure/

wen0x0 committed:
Update wen.json

I have updated the record key from cname to CNAME as per the naming convention.
Please review when you have time. Thank you for your support!
